src/background/index.ts

```diff
@@ -253,6 +306,21 @@ const messageListener = (request) => {
       // Send a message to the active tab
       agent.Primers.updateLogs(request.primer.slug, request.log.id, request.remove).then(response => {
         chrome.tabs.sendMessage(activeTab.id, {message: "update_primer_response", body: response });
+
+        if (response.removed) {
+          window.analytics.track('Log Removed', {
+            id: response.log_id,
+            contentId: response.content_id,
+            context: 'midas'
```

Comment:
FYI primerId not included in response, so there's no way of me tracking that here

Reply:
You actually have access to the id from the request here (see line 307, `request.primer.slug`) I believe you can call request.primer.id  here.